samcli/commands/check/bottle_necks.py

```diff
@@ -0,0 +1,72 @@
+"""
+Bottle neck questions are asked here. Data is saved in graph, but not calcualted here.
+"""
+import click
+
+from samcli.commands.check.resources.LambdaFunction import LambdaFunction
+from samcli.commands._utils.resources import AWS_LAMBDA_FUNCTION
+
+
+class BottleNecks:
+    def __init__(self, graph):
+        self._graph = graph
+        self._lambda_max_duration = 900000
+
+    def ask_entry_point_question(self):
+        entry_points = self._graph.get_entry_points()
+
+        # All entry points must be calcualted before info can be displayed
+        while entry_points:
```

Comment:
Looking at this `while` block, it seems user has to answer all questions, regardless of the order of questions. Two questions:
1. Does the order of questions answer matter the end result?
2. Will there be any way for user to skip answer some questions?

Reply:
1. Yes, the order does matter. Right now it technically doesn't matter, but once more resources are added, it will matter.
2. No. Calculations (both bottle neck and pricing) cannot be done unless all questions are answered.